index.html

```diff
@@ -0,0 +1,42 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Document</title>
+    <link rel="stylesheet" href="styles.css">
+</head>
+<body>
+    <nav>
+        <img src="/Images/Light.png"/>
+    </nav>
+    <header class="bar">
+        <img src="/Images/app_bar.png"/>
+    </header>
+    <div class="container">
+        <section class="head">
+            <h2> 
+                Hey there. I’m Santiago. 
+                <div class="title"> I’m a software developer </div>
+            </h2>
+        </section>
+        <section class="text">
+            <p>
+                I can help you build a product , feature or website Look through some of my work and experience! If you like what you see and have a project you need coded, 
+                don’t hesitate to contact me.
+            </p>
+        </section>
+        <div class="socials">
+            <a href="https://twitter.com/NoizeSalt"> <img src="Images/twitter.png" alt="Twitter"/> </a>
+            <a href="https://www.linkedin.com/in/santiago-ruido-a1404880/"> <img src="Images/Linkedin icon.png" alt="LinkedIn"/> </a>
+            <a href="https://medium.com/"> <img src="Images/medium.png" alt="Medium"/> </a>
+            <a href="https://github.com/Arch-Noize"> <img src="Images/github.png" alt="GitHub"/> </a>
+            <a href="https://www.angellist.com/"> <img src="Images/angellist.png" alt="Angellist"/> </a>
+        </div>
```

Comment:
- [ ] I think it would be better to use `ul` tag to group, and `li` to list eash icon As suggested by the previous reviewer too 👍.

   The `<ul>` tag in HTML is used to create bulleted lists. It provides semantic structure, improves readability and organization, enhances accessibility, allows for customization through CSS, and promotes maintainability by separating content structure from presentation. In summary, the `<ul>` tag is valuable for creating unordered lists with clear visual cues and meaningful structure. 👍 👍 

- [ ] **[Optional]** It would be better also to adjust the links of the icons as they are not visible on my side, Kindly check this for the best user experience 👍

Reply:
I don't mind using the ul tag, but it would make a bulleted list when the template in question doesn't uses it and the activity doesn't require it.

I can perfectly add it but I just want to be on record that it will differ from the template's design 

Also, odd that the images don't load for you. Will see what's going on there

Reply:
@Arch-Noize, It is possible to remove the bullets in CSS by adding this line `list-style-type: none ;` This will remove the bullets 😎